scripts/gitlab/build_and_test.sh

```diff
@@ -253,7 +253,15 @@ then
     # make clean
 fi
 
+cd ${project_dir}
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 echo "~~~~~ Build and test completed"
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+echo "~~~~~ To reproduce this build:"
+echo ""
+echo " #`git rev-parse HEAD` on host ${truehostname}"
+echo ""
+echo "SPACK_DISABLE_LOCAL_CONFIG=\"\" SPACK_USER_CACHE_PATH=\"ci_spack_cache\" python3 scripts/uberenv/uberenv.py \"${spec}\""
```

Comment:
The "--prefix" option needs to be added in order for the subsequent cmake to work.  I would suggest the following change:
```suggestion
echo "SPACK_DISABLE_LOCAL_CONFIG=\"\" SPACK_USER_CACHE_PATH=\"ci_spack_cache\" python3 scripts/uberenv/uberenv.py \"${spec}\" --prefix=\"../ci_uberenv\""
```